provider/ec2/internal/ec2instancetypes/process_cost_data.go

```diff
@@ -179,6 +179,11 @@ func process(in io.Reader) (map[string][]instanceType, metadata, error) {
 		if productInfo.Tenancy != "Shared" {
 			continue
 		}
+		if productInfo.PreInstalledSW != "NA" {
```

Comment:
couldn't it also be "" ?

Reply:
Unfortunately not according to the `index.json`

```
"preInstalledSw" : "NA",
```
vs
```
 "preInstalledSw" : "SQL Ent",
```